virtcontainers/container.go

```diff
@@ -51,6 +51,10 @@ var cdromMajors = map[int64]string{
 	32: "CM206_CDROM_MAJOR",
 }
 
+// https://github.com/torvalds/linux/blob/master/include/uapi/linux/major.h
+// #FLOPPY_MAJOR		2
+var floppyMajor = int64(2)
```

Comment:
@jodh-intel Made this a const.
I did not add it to the map in this case, just so that we can clearly distinguish them and pass this in the log mesage.